src/CMakeLists.txt

```diff
@@ -87,11 +87,9 @@ if (MMAL_FOUND)
 endif()
 
 find_package(FFMPEG)
-if (FFMPEG_FOUND)
-    message(STATUS "LIBAV Supported")
-    target_include_directories(ada PUBLIC ${FFMPEG_INCLUDE_DIRS})
-    target_link_libraries(ada PUBLIC ${FFMPEG_LIBRARIES})
-    target_compile_definitions(ada PUBLIC SUPPORT_LIBAV )
```

Comment:
You will need `target_compile_definitions(ada PUBLIC SUPPORT_LIBAV )`. The flag is required to compile TextureStreamAv https://github.com/patriciogonzalezvivo/ada/blob/main/src/gl/textureStreamAV.cpp#L6

Reply:
I can just rename FFMPEG back to LIBAV, the doc says that calling `target_link_libraries`.

Reply:
Sorry I don't follow, `SUPPORT_LIBAV` it's just define flag. If you remove it,  `TextureStreamAV` is not compile ( https://github.com/patriciogonzalezvivo/ada/blob/main/src/gl/textureStreamAV.cpp#L6 )